ROMFS/px4fmu_common/mixers/quad_x_cw.main.mix

```diff
@@ -2,4 +2,3 @@
 # Quad X with clock-wise motor assigment
 
 R: 4xcw 10000 10000 10000 0
-
```

Comment:
Attention, this actually break the mixer load. It's quite weird I know!

Reply:
Thanks for reviewing @julianoes !  I'm not sure why these changes show up in the browser without the newline character at the last line in each file, (at the end of line 4 in this comment), but I've checked this file and all others and there is one newline in each file as there needs to be.  This PR only removes duplicated newline characters in the affected files.

Reply:
Aha, I suggest to test if the mixer works before merging then.

Reply:
Thanks @julianoes , 

From this PR:
`INFO  [init] Mixer: /etc/mixers/quad_x_cw.main.mix on /dev/pwm_output0
`

Reply:
I'm looking at the mixer files and they no longer have an extra newline at the end.
As far as I know they would not work anymore.

Reply:
It does work in SITL however, so I might be wrong.

Reply:
I would try to check that with a real airframe as well and see if the motors still work. And it needs to be an airframe which does not have additional mixers after the multicopter mixer because the newline would only affect the last mixer line.

Reply:
Hi @julianoes , the mixer files all still need (and have) one newline character, however github doesn't seem to show them through the browser.  Only one mixer file, the [quad_x_cw.main.mix](https://github.com/PX4/Firmware/blob/b761060c9b979f372d019cec389930a2c7765552/ROMFS/px4fmu_common/mixers/quad_x_cw.main.mix#L5) file, has 2 newline characters at present.

Here is what the current master branch looks like in a text editor:
![image](https://user-images.githubusercontent.com/2497951/79357419-3b9a1200-7efd-11ea-8f7a-b415e2f08c51.png)

And here is another representative file:
![image](https://user-images.githubusercontent.com/2497951/79357731-a51a2080-7efd-11ea-9bef-88108c3e8ec3.png)


If you open them in a browser you can also see there is a difference in the blank line displayed in quad_x_cw.main.mix and not in any other file.

Hopefully this makes sense and my apologies for this wasting some of your time. :}